client/app/bundles/course/assessment/submission/components/AnswersHistory.jsx

```diff
@@ -1,12 +1,12 @@
 import React, { Component } from 'react';
 
-import ProgrammingAnswerHistory from '../containers/ProgrammingAnswerHistory';
+import PastAnswers from '../containers/PastAnswers';
 
 // Abstraction layer for future answer history components of different answer types
 export default class AnswersHistory extends Component {
-  static renderProgramming(question) {
+  static renderPastAnswers(question) {
```

Comment:
Yeah ok.
In order to do away with AnswersHistory I'll have to find a way to convert PastAnswers to a function instead of class. This is because the parent component SubmissionAnswer requires `getRenderer` to return a function.